modules/artifact/src/main/java/org/mule/module/artifact/classloader/JdbcResourceReleaser.java

```diff
@@ -44,7 +48,11 @@
  */
 public class JdbcResourceReleaser implements ResourceReleaser {
 
+  private static final String avoidThreadDisposal = getProperty("avoid.dispose.oracle.threads");
```

Comment:
use Boolean.getBoolean which verifies if there exists a system property, otherwise it assigns a default.
Use a constant for the string and rename this as jdb.resource.releaser.avoid.dispose.oracle.threads